operator-common/src/test/java/io/strimzi/operator/common/operator/resource/AbstractCustomResourceOperatorIT.java

```diff
@@ -233,13 +234,13 @@ public void testUpdateStatusAfterResourceUpdatedThrowsKubernetesClientException(
                     LOGGER.info("Updating resource status after underlying resource has changed");
                     return op.updateStatusAsync(Reconciliation.DUMMY_RECONCILIATION, newStatus);
                 })
-                .onComplete(context.failing(e -> context.verify(() -> {
-                    assertThat("Exception was not KubernetesClientException, it was : " + e.toString(),
-                            e, instanceOf(KubernetesClientException.class));
-                    updateFailed.complete();
+                .onComplete(context.succeeding(res -> context.verify(() -> {
+                    assertThat(res.getMetadata().getName(), is(resourceName));
+                    assertThat(res.getMetadata().getNamespace(), is(namespace));
+                    updateStatus.complete();
```

Comment:
Does this mean that it fixes #5326?

Reply:
I would need to elaborate. But when the resource & status is modified simultaneously, it does not throw.

Reply:
Well, the exception you removed in the test seems to be what #5326 complains about, or?

Reply:
Yes, you are right.